default.nix

```diff
@@ -6,21 +6,27 @@ args@{ system ? builtins.currentSystem
 }:
 with obelisk;
 let
+  # All the versions that the user cares about are here so that they can
+  # be changed in one place
+  chainWeaverVersion = "1.0";
```

Comment:
Huh never saw the `w` in `Chainweaver` capitalized

Reply:
Yeah, good point. I'll update. I didn't do this out of any reason. Just the first thing I did and autocomplete did the rest, haha.